src/ontology/mondo-edit.obo

```diff
@@ -263186,18 +263186,19 @@ property_value: confidence "3.533333333333334" xsd:double
 
 [Term]
 id: MONDO:0013209
-name: non-alcoholic fatty liver disease
-def: "A term referring to fatty replacement of the hepatic parenchyma which is not related to alcohol use." [NCIT:C84444]
+name: metabolic dysfunction-associated steatotic liver disease
+def: "Metabolic dysfunction-associated steatotic liver disease (MASLD, formerly known as nonalcoholic fatty liver disease or NAFLD) is a type of liver disease that is not caused by alcohol. It typically does not cause symptoms in the early stages, but it can cause health problems due to fat accumulation, inflammation, and scarring in the liver." [https://www.verywellhealth.com/how-dangerous-is-non-alcoholic-fatty-liver-disease-2223390]
 subset: predisposition
 synonym: "fatty liver disease, nonalcoholic" EXACT [MONDO:0000031]
 synonym: "fatty liver disease, nonalcoholic, susceptibility to, 1" RELATED [MONDO:Lexical, OMIM:613282]
 synonym: "liver disease, alcoholic, susceptibility to, 1" RELATED [OMIM:613282]
+synonym: "MASLD" EXACT ABBREVIATION [https://www.verywellhealth.com/how-dangerous-is-non-alcoholic-fatty-liver-disease-2223390]
 synonym: "NAFLD" EXACT ABBREVIATION [Orphanet:33271]
-synonym: "NAFLD - nonalcoholic fatty liver disease" RELATED [NCIT:C84444]
+synonym: "NAFLD - nonalcoholic fatty liver disease" EXACT [NCIT:C84444]
 synonym: "NAFLD1" RELATED ABBREVIATION [MONDO:Lexical, OMIM:613282]
 synonym: "non-alcoholic fatty liver" EXACT [MONDO:0007026]
 synonym: "non-alcoholic fatty liver disease" EXACT [DOID:0080208]
-synonym: "nonalcoholic fatty liver disease" RELATED [NCIT:C84444]
+synonym: "nonalcoholic fatty liver disease" EXACT [NCIT:C84444]
```

Comment:
This synonym is duplicated with the DOID one

Reply:
I'm not sure what you mean? The DOID syn has is 'non-alcoholic' vs the NCIT is 'nonalcoholic' (no dash). Am I missing something?

Reply:
My bad! My brain ignored the dash (and I checked several times!!! Obviously, I should not be reviewing PR -hahaha!

all is well then. This is ready to merge.